asm/macros/event.inc

```diff
@@ -2321,3 +2333,64 @@
 	.macro togglefakertc
 	callnative Script_ToggleFakeRtc
 	.endm
+
+	@ Makes the trainer unable to see the player if executed.
+	@ This is a no-op if the player interacts with the trainer.
+	.macro cant_see_if, condition:req
+	callnative Script_EndTrainerCanSeeIf, player_visible=FALSE
+	.byte \condition
+	.endm
+
+	.macro cant_see
+	cant_see_if_unset 0
```

Comment:
Any chance we could replace these magic numbers (the ones down below too) with constants, too? I'm aware external #defines don't work (yet, hopefully), but perhaps a ``.set`` could provide more clarity than magic numbers

Reply:
This particular magic number is just a flag that's always `FALSE` (flag 0).

The other magic numbers come from `goto_if_*`/`call_if_*`. I suppose we could PR them to pret, but in some sense they're documented by the names of the macros they're used in and have no other use.